src/App/Pages/Generator/GeneratorPage.xaml

```diff
@@ -152,8 +175,7 @@
                             HorizontalOptions="StartAndExpand" />
                         <Switch
                             IsToggled="{Binding Uppercase}"
-                            IsEnabled="{Binding EnforcedPolicyOptions.UseUppercase,
-                                Converter={StaticResource inverseBool}}"
+                            IsEnabled="{Binding IsUppercaseSwitchEnabled}"
```

Comment:
@mportune-bw What prompted this change? This is how I thought you would have had to have done it fromt he beginning when I questioned it the first time.

Reply:
Hm I thought there was a scenario where this was necessary but now I'm wondering if I just confused myself with the styling of disabled vs off switch states.  I've reverted it locally and everything is still working as it should.  I'll submit a PR shortly.